docs/specifications/1-introduction.md

```diff
@@ -0,0 +1,172 @@
+# Introduction
+
+This document provides a high level overview of *jsii*, starting with it's
+design tenets. It introduces the concepts and components that compose *jsii*.
+
+
+## Updating the Specification
+
+### Introduction
+
+The *jsii* specification follows the guiding principles of an RFC. It is a
+living document that describes the current understanding of how the various
+[components](#components) of *jsii* are operating, as well as the approaches
+used to ensure consistent behavior across the various supported languages.
+
+The document is mastered with the *jsii* codebase, making it easy to determine
+what specification was in place at the time of a give *jsii* release (by ways of
+referring to the `vX.Y.Z` git tag). A single version of the specification is
+considered **active** at any given time: the version of the specification that
+is represented on the `HEAD` commit of the `master` branch of the [`aws/jsii`]
+repository. The **active** specification must be the base version for any update
+proposal.
+
+[`aws/jsii`]: https://github.com/aws/jsii
+
+The process to update the specification is intended to be as lightweight as
+possible, while ensuring sufficient conversation takes place before implementing
+significant (and breaking) changes. Since the process to update the
+specification is part of the specification itself, it is amenable to be changed
+following the process described in the currently **active** specification.
+
+### Process
+
+While the general process for updating the specification is to create a GitHub
+pull request against the [`aws/jsii`] repository, the exact requirements for
+what should be included in the pull request vary depending on the type of update
+that is proposed:
+
+- [:warning: Changing Behavior](#new-behavior) describes the process to be
+  followed when introducing changes to the behavior of any component of *jsii*:
+  new features, breaking changes to existing features, ...
+- [:mag: Addressing Gaps](#addressing-gaps) is the process used for adding
+  specification around existing but un-specified behavior.
+- [:thumbsup: Trivial Changes](#trivial) explains how to propose changes that
+  improve the specification without changing it's meaning.
+
+#### <a id="new-behavior"/> :warning: Changing Behavior
```

Comment:
Why isn't this called "Breaking Change"?

Reply:
Because a new feature isn't necessarily a breaking change, but most often also should be handled through RFC (be it only to make sure the feature does not paint us into a corner).